official/utils/logs/logger.py

```diff
@@ -30,6 +30,7 @@
 import threading
 import uuid
 
+import requests
```

Comment:
Isn't this a builtin? Why not with the builtins above?

Reply:
Its actually not a builtin. I was bite by lint for this.